src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerTeamController.cs

```diff
@@ -332,11 +332,20 @@ public ActionResult Row1Panel1(AccountDashboardViewModel model)
         [ChildActionOnly]
         public ActionResult Row1Panel2(AccountDashboardViewModel model)
         {
-            var viewModel = new PanelViewModel<AccountDashboardViewModel> { ViewName = "ProviderPermissions", Data = model };
+            var viewModel = new PanelViewModel<AccountDashboardViewModel> { Data = model };
             if (model.PayeSchemeCount == 0 || model.AgreementsToSign)
             {
                 viewModel.ViewName = "ProviderPermissionsDenied";
             }
+            else if (model.AccountViewModel?.Providers?.Any() ?? false)
```

Comment:
The model should provide a flag to determine the state to display.  Please move this logic to the model.

Reply:
The logic has been moved into the model. We can introduce a flag if the property is called more than once.